samples/drivers/display/src/main.c

```diff
@@ -165,6 +165,31 @@ static inline void fill_buffer_l_8(enum corner corner, uint8_t grey, uint8_t *bu
 	}
 }
 
+static void fill_buffer_al_88(enum corner corner, uint8_t grey, uint8_t *buf,
+				 size_t buf_size)
+{
+	uint16_t color = 0;
```

Comment:
Initialization not needed

Reply:
Doesn't hurt either, as the switch has no default case

Reply:
> Doesn't hurt either, as the switch has no default case

well, it should :)
https://docs.zephyrproject.org/latest/contribute/coding_guidelines/index.html#:~:text=Every%20switch%20statement%20shall%20have%20a%20default%20label